assets/src/js/main.js

```diff
@@ -1,14 +1,10 @@
 /**
  * Main scripts, loaded on all pages.
- *
- * @package Blank-Theme
  */
 
 import '../sass/main.scss';
 import * as components from './components';
```

Comment:
Instead of `*` can we add it like,

```javascript
import { WebFont, common  } as components from './components';
```

And while init use below code,

```javascript
WebFont.init();
common.init();
```

**Reason:**

I observed in multiple projects that the same line gets copied on `single.js` and `home.js` and it loads all components in all these files. I agreed that it's not getting fire on those pages but it increasing the file size unnecessary.

Reply:
Good Catch, that's how it should be.